rakelib/form526_submission_submit_endpoint_update.rake

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+namespace :form526 do
```

Comment:
Nit: if these are going to be namespaced and we would put more here I'd say either create a nested folder in `rakelib` for form 526 or rename the filename here to be more generic

Reply:
File will be destroyed after the task is run so I'm not concerned about future conflicts or misrepresentation.